<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MONO-HYD-LAP-12" name="readOTP.OTP_Verification" tests="1" failures="0" timestamp="25 Oct 2019 07:49:39 GMT" time="2.698" errors="1">
  <testcase name="barCode" time="2.698" classname="readOTP.OTP_Verification">
    <error type="com.twilio.exception.ApiException" message="The requested resource /Sessions/ACc49193614eaaf903351d981f58a32c33/Messages was not found">
      <![CDATA[com.twilio.exception.ApiException: The requested resource /Sessions/ACc49193614eaaf903351d981f58a32c33/Messages was not found
	at com.twilio.rest.messaging.v1.session.MessageReader.pageForRequest(MessageReader.java:150)
	at com.twilio.rest.messaging.v1.session.MessageReader.firstPage(MessageReader.java:67)
	at com.twilio.rest.messaging.v1.session.MessageReader.read(MessageReader.java:47)
	at com.twilio.base.Reader.read(Reader.java:25)
	at readOTP.OTP_Verification.barCode(OTP_Verification.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- barCode -->
</testsuite> <!-- readOTP.OTP_Verification -->
